app/SmartAlbums/Utils/MimicModel.php

```diff
@@ -62,7 +62,7 @@ public function toJson($options = 0): string
 	 */
 	public function __get(string $key)
 	{
-		if (empty($key)) {
+		if ($key != '') {
```

Comment:
```suggestion
		if ($key !== '') {
```
Or would that change the behavior here compared to the original `empty`?

Reply:
not sure, but it didn't like empty() because it is a bit too lose in comparison.